File: FindNthPrime.js
class FindNthPrime {
	static find (n) {
		let primeCount = 0;
		let currentNumber = 0;
		while(primeCount < n) {
			currentNumber++;
			if(FindNthPrime.isPrime(currentNumber)) {
				//console.log(currentNumber);
				primeCount++;
			}
		}
		return currentNumber;
	}

	static isPrime(candidate) {
	for (let divisor = 2; divisor < candidate - 1; divisor++) {
		if(candidate % divisor == 0) {
			return false;
		}
	}
		return true;
	}
}

module.exports = FindNthPrime;

File: src/main/java/FindNthPrime.java
public class FindNthPrime {
	public static int find(int n) {
		int primeCount = 0;
		int currentNumber = 0;
		while(primeCount < n) {
			currentNumber++;
			if(isPrime(currentNumber)) {
//				System.out.println(currentNumber);
				primeCount++;
			}
		}
		return currentNumber;
	}

	private static boolean isPrime(int candidate) {
		for (int divisor = 2; divisor < candidate -1; divisor++) {
			if(candidate % divisor == 0) {
				return false;
			}
		}
		return true;
	}
}
